cmd/triage/main.go

```diff
@@ -9,6 +9,7 @@ import (
 	"sync"
 
 	jira "github.com/andygrunwald/go-jira"
+	"github.com/shiftstack/bugwatcher/cmd/triage/tasker"
```

Comment:
Could we use [`sync.Map`][1] here instead of writing our own variant? I'm not sure if they achieve the same thing or not but it looks like they do?

[1]: https://pkg.go.dev/sync#Map

Reply:
Sure! I'll leave that to you.

Reply:
Hmm, seems `sync.Map` isn't really designed for this. More for read-heavy workloads. TIL.